db/migrate/20241014031756_add_deleted_at_to_course_users.rb

```diff
@@ -0,0 +1,6 @@
+class AddDeletedAtToCourseUsers < ActiveRecord::Migration[7.0]
+  def change
+    add_column :course_users, :deleted_at, :datetime
```

Comment:
Can be done in a single line: `add_column ... index: true`  
Separately, can explicity specify `default: nil`?

Also can put everything in a single migration file. Makes it easier for feature rollback if need be.

Reply:
Actually, this is the generated file after running `rails db:migrate`. Can we skip it ?

Reply:
How does `rails db:migrate` generate the migration file?  
Did you mean `rails generate migration xxx`?

Reply:
Yes, you are correct.

Reply:
After generating the migration files, any time before executing the migration with `rails db:migrate`, you can edit the migration file yourself.

Reply:
Ok, I updated for all migration files.